tests/queries/0_stateless/02983_const_sharding_key.sql

```diff
@@ -0,0 +1,24 @@
+-- Tags: distributed, no-parallel
+
+DROP DATABASE IF EXISTS shard_0;
+DROP DATABASE IF EXISTS shard_1;
+DROP TABLE IF EXISTS t_distr;
+
+CREATE DATABASE IF NOT EXISTS shard_0;
+CREATE DATABASE IF NOT EXISTS shard_1;
+
+CREATE TABLE shard_0.t_local (a Int) ENGINE = Memory;
+CREATE TABLE shard_1.t_local (a Int) ENGINE = Memory;
+CREATE TABLE t_distr (a Int) ENGINE = Distributed(test_cluster_two_shards_different_databases, '', 't_local', 1000);
```

Comment:
A constant sharding key is not very useful but still valid. We must not crash on it.